target/linux/ath79/image/tiny-tp-link.mk

```diff
@@ -1,5 +1,44 @@
 include ./common-tp-link.mk
 
+define Device/tplink_rex5x-v1
+  $(Device/tplink-safeloader)
+  SOC := qca9558
+  IMAGE_SIZE := 6016k
+  DEVICE_PACKAGES := kmod-ath10k-ct-smallbuffers ath10k-firmware-qca988x-ct
+  DEFAULT := n
+endef
+
+define Device/tplink_re355-v1
+  $(Device/tplink_rex5x-v1)
+  DEVICE_MODEL := RE355
+  DEVICE_VARIANT := v1
+  TPLINK_BOARD_ID := RE355
+  SUPPORTED_DEVICES += re355
+endef
+TARGET_DEVICES += tplink_re355-v1
+
+define Device/tplink_re450-v1
+  $(Device/tplink_rex5x-v1)
+  DEVICE_MODEL := RE450
+  DEVICE_VARIANT := v1
+  TPLINK_BOARD_ID := RE450
+  SUPPORTED_DEVICES += re450
+endef
+TARGET_DEVICES += tplink_re450-v1
+
+define Device/tplink_re450-v2
+  $(Device/tplink-safeloader)
+  SOC := qca9563
+  IMAGE_SIZE := 6016k
+  DEVICE_MODEL := RE450
+  DEVICE_VARIANT := v2
+  DEVICE_PACKAGES := kmod-ath10k-ct-smallbuffers ath10k-firmware-qca988x-ct
```

Comment:
Maybe we can remove ath10k package to get more free spaces? User can manually select them on they custom build. In this way, we can get an official build image with 2 GHz WiFi only. Let's wait other reviewer's comment.